frontend/src/container/Tracim.jsx

```diff
@@ -499,8 +498,8 @@ export class Tracim extends React.Component {
 
                 <Route
                   path={PAGE.WORKSPACE.RECENT_ACTIVITIES(':idws')}
-                  render={(routerProps) => (
-                    <WorkspaceRecentActivities workspaceId={routerProps.match.params.idws} />
+                  render={({ match }) => (
```

Comment:
I would like a comment for deprecated route since nothing will now point to this route

Reply:
fix